### DISCUSSION TITLE
cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details

### DISCUSSION THREAD

**Discussion by akay73** [community/community#143660] [Programming Help]
### Select Topic Area

Question

### Body

I'm getting this error in Appium and I couldn't solve it, can you please help?

Thread ended with message "unknown error: 'GET /wd/hub/session/1c70d10e-7fb5-43e6-8d42-b68f070dccaa/source' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details

**Comment by akay73** [comment]
@mykola-mokhnach

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @akay73 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
